tests/ops/util/test_consent_util.py

```diff
@@ -461,6 +479,505 @@ def test_get_fides_user_device_id_provided_identity(
+        assert new_history.disabled is True
+        assert new_history.has_gpc_flag is False
+        assert new_history.displayed_in_privacy_center is True
+        assert new_history.displayed_in_overlay is False
+        assert new_history.displayed_in_api is False
+        assert new_history.version == 1.0
+        assert new_history.id != notice.id
+        assert new_history.version == 1.0
+
+        # Existing notice A - assert this wasn't updated.
+        db.refresh(notice)
+        assert notice.name == "Test Privacy Notice"
+        assert notice.description == "This website uses cookies."
+        assert (
+            unescape(notice.internal_description)
+            == "This is a contrived template for testing.  This field's for internal testing!"
+        )
+        assert notice.regions == [PrivacyNoticeRegion.us_ak]
+        assert notice.consent_mechanism == ConsentMechanism.opt_in
+        assert notice.enforcement_level == EnforcementLevel.system_wide
+        assert notice.disabled is False
+        assert notice.has_gpc_flag is True
+        assert notice.displayed_in_privacy_center is False
+        assert notice.displayed_in_overlay is True
+        assert notice.displayed_in_api is False
+        assert notice.version == 1.0
+        assert notice.histories.count() == 1
+
+        db.refresh(history)
+        # Existing history B - assert this wasn't updated.
+        assert history.name == "Test Privacy Notice"
+        assert history.description == "This website uses cookies."
+        assert (
+            unescape(history.internal_description)
+            == "This is a contrived template for testing.  This field's for internal testing!"
+        )
+        assert history.regions == [PrivacyNoticeRegion.us_ak]
+        assert history.consent_mechanism == ConsentMechanism.opt_in
+        assert history.enforcement_level == EnforcementLevel.system_wide
+        assert history.disabled is False
+        assert history.has_gpc_flag is True
+        assert history.displayed_in_privacy_center is False
+        assert history.displayed_in_overlay is True
+        assert history.displayed_in_api is False
+        assert history.version == 1.0
+        assert history.id != new_history.id
+
+        with pytest.raises(HTTPException):
+            load_default_notices_on_startup(
+                db, "tests/fixtures/test_bad_privacy_notices_update.yml"
+            )
+
+        new_history.delete(db)
+        history.delete(db)
+
+        new_privacy_notice.delete(db)
+        notice.delete(db)
+
+        new_template.delete(db)
+        template.delete(db)
+
+    def test_load_actual_default_notices(self, db):
+        """Sanity check, makings sure that default privacy notices don't load with errors"""
+        new_templates, new_privacy_notices = load_default_notices_on_startup(
+            db, DEFAULT_PRIVACY_NOTICES_PATH
+        )
+        assert len(new_templates) == 5
```

Comment:
nit: i feel like whenever we do this sort of len comparison in our tests, e.g. with saas templates, it ends up being really cumbersome and annoying to keep updated as that default list changes. this one's a bit simpler, but still - may just be worth `assert`ing that `len(new_templates) >= 1` and `len(new_privacy_notices) >= 1`?

Reply:
that's fair. That's also a big reason why I'm not loading these out of the box notices for tests too.